src/components/apiComponents/apiComponentWrapper.jsx

```diff
@@ -118,19 +142,42 @@ const ApiComponentWrapper = (args) => {
     setResponce(null)
     setIsLoading(true)
 
-    const res = await fetch(activeEndpoint.uri, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify(body),
-    })
+    try {
+      // Validate URL
+      // Validate URL
```

Comment:
nit: duplicated comment
```suggestion
      // Validate URL
```